packages/generator/tests/conftest.py

```diff
@@ -37,3 +41,19 @@ def discovery_service_process() -> Generator[DiscoveryServiceProcess, None, None
 def pin_map_directory(test_assets_directory: pathlib.Path) -> pathlib.Path:
     """Test fixture that returns the pin map directory."""
     return test_assets_directory / "pin_map"
+
+
+@pytest.fixture(scope="session")
+def create_client() -> Generator[functools.partial[Result], Any, None]:
```

Comment:
`functools.partial[Result]` is implementation-specific and it's kind of a weak type hint, because it doesn't describe the allowed arguments. I would prefer to annotate this as `Callable[[Sequence[str]],Result]` or whatever.

Reply:
Sure, I have updated to `Callable[[Sequence[str]],Result]`.